src/notation/internal/excerptnotation.cpp

```diff
@@ -82,34 +78,6 @@ bool ExcerptNotation::isEmpty() const
     return m_excerpt->parts().empty();
 }
 
-void ExcerptNotation::fillWithDefaultInfo()
-{
-    TRACEFUNC;
-
-    IF_ASSERT_FAILED(m_excerpt || m_excerpt->excerptScore()) {
```

Comment:
By the way, totally off-topic and irrelevant, but thanks to `clang-analyzer` I noticed that this assert contains a funny mistake, that isn't intuitively obviously because of the negation in `IF_ASSERT_FAILED`. Namely: what happens if `m_excerpt` is `false`?